www/core/test/extension_test.xml

```diff
@@ -23,17 +23,17 @@
 		<description>Joomla! 3.10 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>3.10.10</version>
-		<infourl title="Joomla 3.10.10 Release">https://github.com/joomla/joomla-cms/releases/tag/3.10.10</infourl>
+		<version>3.10.11</version>
+		<infourl title="Joomla 3.10.11 Release">https://github.com/joomla/joomla-cms/releases/tag/3.10.11</infourl>
```

Comment:
Also this should go to the finale release URL please

Reply:
Is the final release URL different for 3.10.11 compared to 3.10.10?